tests/unit/sim_proxy/proxy_simulator_controls_test.py

```diff
@@ -37,7 +38,7 @@
 with open(TEST_SECTOR, "r") as f:
     geojson = json.load(f)
     del geojson["_source"]
-    sector_element = SectorElement.deserialise(json.dumps(geojson))
+    sector_element = SectorElement.deserialise(StringIO(json.dumps(geojson)))
```

Comment:
@rkm Sorry - coming to this late, but do we need to dump and read from a string here? 

It should be possible to pass the `geojson` from line 39 directly to `SectorElement.deserialise` (as in `sector_test.py` above). If not I need to improve the `deserialise` method.

I think this applies to `sector_validation.py` (above) too.

Reply:
You're right, but I added a "_source" tag to the test files which references the git hash in Aviary where they were copied from. I *thought* this was sensible to do at the time, although not so sure anymore!